.eslintignore

```diff
@@ -0,0 +1,4 @@
+spec/fixtures/**/*.js
+node_modules
```

Comment:
`node_modules` is [ignored by default](https://eslint.org/docs/user-guide/configuring#eslintignore) in ESLint. (This _may_ be required for `prettier` though?)

Reply:
`eslint` by default only ignore the top-level `node_modules`, so I needed to add this one (which ignores any nested one) to ignore `script/node_modules`

Reply:
Interesting, I didn't know that... but it's right in the documentation as an important note 🤦‍♂.